src/java/org/apache/cassandra/index/sai/metrics/ColumnQueryMetrics.java

```diff
@@ -70,6 +70,61 @@ public QueryEventListener.PostingListEventListener postingListEventListener()
         }
     }
 
+    public static class BKDIndexMetrics extends ColumnQueryMetrics implements QueryEventListener.BKDIndexEventListener
```

Comment:
nit: Just another TODO where we might have to do some slight renaming later. Heh, if it becomes like `BalancedBlockTree` this could become `BBT` or something :p